### PR TITLE
Prevent image cropping, unify publication layout, fix strange margin behavior

### PR DESCRIPTION
**Pull request by AndreasMadsen**
Screenshot 2024-05-17 at 12 06 05
Screenshot 2024-05-17 at 12 04 44

### CODE REVIEW

**Comment by xhluca** [comment]
For mobile view can it be a bit wider? on phone screen some of the images might be harder to read.

**Comment by AndreasMadsen** [comment]
I think there is a misunderstanding about how big phone screens are. Here is a simulation.

Screenshot 2024-05-17 at 18 17 46

**Comment by xhluca** [comment]
I realized the image is now below the title and above the links. Is it possible to keep the links above the image? It feels visually a bit odd IMO.

File: _sass/custom/display-publications.scss [hunk 1]
```diff
@@ -1,23 +1,47 @@
...
+        "thumbnail links"
+        "thumbnail .";
+    grid-template-columns: 300px 1fr;
+
+    @media (max-width: 600px) {
+        grid-template-areas:
+        "title"
+        "authors"
+        "venue"
+        "thumbnail"
```
Comment (AndreasMadsen) [inline _sass/custom/display-publications.scss:16]: Just change the order here.

Comment (xhluca) [reply _sass/custom/display-publications.scss]: Done

**Pull request merged**